nextflow.config

```diff
@@ -0,0 +1,130 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    UMCUGenetics/GIABEval Nextflow config file
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+*/
+
+manifest {
+    name            = 'UMCUGenetics/GIABEval'
+    author          = """UMCU Genetics"""
+    homePage        = 'https://github.com/UMCUGenetics/GIABEval'
+    description     = """UMCU Genetics Genome-in-a-bottle evaluation workflow for benchmarking"""
+    mainScript      = 'main.nf'
+    nextflowVersion = '>=23.10.1'
+    version         = '1.0dev'
+    doi             = ''
+}
+
+// Nextflow plugins
+plugins {
+    id 'nf-validation' // Validation of pipeline parameters and creation of an input channel from a sample sheet
+}
+
+// Global default params, used in configs
+params {
+    // Filename settings
+    delim = "_"
+    id_index = [0,3]
+ 
+    // Input options
+    vcf_path = null
+    nist_version_to_use = "hg001_nist_v2_19_grch37"  // should match with defined params
```

Comment:
If this should match with the defined params, why not use the parameter?

Reply:
Because there can be more benchmarking references in the config.
Using param nist_version_to_use could be overwritten when starting the workflow: wf can thus be started for different GIAB versions (including sample, version, reference genome version)